### PR TITLE
release 10.14

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-14-0.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+---
+
+- Fixed an issue related to on-demand pagination in Datatable, which disappears when all the rows are expanded and sorted by any displayed column.
+- Fixed an issue related to Datatable `Header Onclick` event not triggering.
+- Fixed an issue related to Veracode Scan in AppRuntimeController.
+- Fixed an issue related to SOAP service is not working in preview showing NoClassDefFoundError.
+- Fixed an alignment issue of Textarea's helptext when used in Form. 
+- Fixed an issue related to Oauth WebService Variable not triggering Success callback in the first call.
+
+### Team Portal
```
Comment (krishnaraoch) [inline learn/wavemaker-release-notes/v10-14-0.md:51]: Not required this. these are fixes during development. regresssion kind of things..
Nothing to mention.

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-14-0.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+:::note
+`wmshowcase` is the WaveMaker Online team's name and WaveMaker Enterprise's name in the URL.
+:::
+
+For more information, see [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment).
+
+## Improvements
+---
+
+- Docker-compose accept env variables with character `_`.
```
Comment (krishnaraoch) [inline learn/wavemaker-release-notes/v10-14-0.md:38]: Who did this ?
Docker-compose accept env variables with character ??

File: learn/wavemaker-release-notes/v10-14-0.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+Old: `http://wmshowcase.cloud.wavemakeronline.com/TestApp`  
+New: `http://cloud.wavemakeronline.com/wmshowcase/TestApp`  
+
+:::note
+`wmshowcase` is the WaveMaker Online team's name and WaveMaker Enterprise's name in the URL.
+:::
+
+For more information, see [Demo Cloud](/learn/app-development/deployment/pipelines-phases#deployment-environment).
+
+## Improvements
```
Comment (krishnaraoch) [inline learn/wavemaker-release-notes/v10-14-0.md:35]: Serach projects by User in Teamportal

**Review by krishnaraoch — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-14-0.md (hunk 2 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-14-0.md:38]: @udaysa

**Review by krishnaraoch — APPROVED** [review]

**Pull request merged**